src/keys-builder/template/structural-directive.extractor.ts

```diff
@@ -28,6 +29,7 @@ import {
 } from './utils';
 
 interface ContainerMetaData {
+  exp?: any;
```

Comment:
Why any?

Reply:
The *exp* property of *ContainerMetaData* is assigned there :

https://github.com/ngneat/transloco-keys-manager/blob/78da210023d359c0331d1fb49448ff443b118bf1/src/keys-builder/template/structural-directive.extractor.ts#L91-L93

I've written a *any* type because `MethodCall['args']` is typed `any[]`. 
We can type that property `AST` but is it safe ? In other way is `MethodCall['args']` always typed as `AST[]` ?

Reply:
You tell me 🙂 
Please add the correct typing, `any` will result in bugs in the future.